atlite/cutout.py

```diff
@@ -42,14 +43,24 @@
 class Cutout(object):
     dataset_module = None
 
-    def __init__(self, name=None, data=None, cutout_dir=".", **cutoutparams):
+    def __init__(self, name=None, data=None, cutout_dir=None, **cutoutparams):
+
         if isinstance(name, xr.Dataset):
             data = name
             name = data.attrs.get("name", "unnamed")
-
-        if '/' in name:
-            cutout_dir, name = os.path.split(name)
-
+        
+        dn = os.path.dirname(name)
+        if dn:
+            name = os.path.basename(name)
```

Comment:
I'd prefer the explicit path in `name` rather than the one in `cutout_dir`.

```python
dirname, name = os.path.split(name)
if dirname:
    cutout_dir = dirname
elif cutout_dir is None:
    if config.cutout_dir:
         cutout_dir = config.cutout_dir
    else:
         cutout_dir = "."
```